### DISCUSSION TITLE
4 click segmentation

### DISCUSSION THREAD

**Discussion by TDH21** [cnr-isti-vclab/TagLab#39] [General]
Hello
I have updated to the latest version of TagLabs and I cannot get the 4 click segmentation to work. When I place the cursor over the icon a red box comes up. As I attempt to use the function nothing happens. The rest of the features of TagLabs work.
I still have the old version 0.6 and the 4 click segmentation works with V 0.6.
Do you have any suggestions to fix this? 
Best, Tim

**Comment by maxcorsini** [comment]
Active the tool. Then, press SHIFT + mouse left button to assign the points at the extreme of the object. When 4 points are assigned the tool automatically segment the object.

**Comment by TDH21** [comment]
Great, that worked. I thought it might be something simple.
Thanks

From: Massimiliano Corsini ***@***.***>
Date: Thursday, 12 May 2022 at 5:01 am
To: cnr-isti-vclab/TagLab ***@***.***>
Cc: TDH21 ***@***.***>, Author ***@***.***>
Subject: Re: [cnr-isti-vclab/TagLab] 4 click segmentation (Discussion #39)

Active the tool. Then, press SHIFT + mouse left button to assign the points at the extreme of the object. When 4 points are assigned the tool automatically segment the object.
